Source/Cogworks.Umbraco.Essentials/Models/ResponsiveImage.cs

```diff
@@ -11,5 +11,11 @@ public class ResponsiveImage
         public string ContainerClass { get; set; }
 
         public string AltText { get; set; }
+
+        public string LazyLoadValue { get; set; }
```

Comment:
Should we rename it to `Loading` or `LoadingSetting` or `LoadingValue` or `LazyLoading` or `[Lazy]LoadingConfiguration`  like on https://github.com/thecogworks/Cogworks.Umbraco.Essentials/blob/168a064a092de44900bbc276a6276dffa6913ed4/Source/Cogworks.Umbraco.Essentials/ViewFiles/_ResponsiveImage.cshtml#L24